.circleci/config.yml

```diff
@@ -60,6 +60,6 @@ jobs:
               echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
               yarn run docs-site
               cd docs/beta
-              yarn install --immutable
+              yarn install
```

Comment:
Turns out not doing `npm ci` was on purpose

Reply:
Maybe put a comment here